content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,243 +5,255 @@ weight = 20
+          IBM Cloud
+        </td>
+        <td>
+          <a href="https://ibm.github.io/manifests/">Website</a>
         </td>
       </tr>
       <tr>
-        <td>Kubeflow on Nutanix</td>
-        <td>Nutanix</td>
         <td>
           Nutanix Kubernetes Engine
         </td>
         <td>
-          <a href="https://nutanix.github.io/kubeflow-manifests">Website</a>
+          {{% nutanix/latest-version %}}
         </td>
         <td>
-          {{% nutanix/latest-version %}}
+          Nutanix
+        </td>
+        <td>
+          <a href="https://nutanix.github.io/kubeflow-manifests">Website</a>
         </td>
       </tr>
       <tr>
-        <td>Charmed Kubeflow</td>
-        <td>Canonical</td>
         <td>
-          All Certified Kubernetes Distributions <sup>[<a href="https://kubernetes.io/partners/#conformance">1</a>]</sup>
+          Amazon Elastic Kubernetes Service (EKS)
         </td>
         <td>
-          <a href="https://charmed-kubeflow.io/">Website</a>
+          {{% aws/latest-version %}} <sup>[<a href="https://github.com/awslabs/kubeflow-manifests/releases">Release Notes</a>]</sup>
         </td>
         <td>
-          1.8.0
+          Amazon Web Services
+        </td>
+        <td>
+          <a href="https://awslabs.github.io/kubeflow-manifests">Website</a>
         </td>
       </tr>
       <tr>
-        <td>deployKF</td>
-        <td>Aranui Solutions</td>
         <td>
-          All Certified Kubernetes Distributions <sup>[<a href="https://kubernetes.io/partners/#conformance">1</a>]</sup>
+          Azure Kubernetes Service (AKS)
         </td>
         <td>
-          <a href="https://www.deploykf.org/">Website</a>
+          {{% azure/latest-version %}} <sup>[<a href="https://github.com/Azure/kubeflow-aks/releases/tag/v{{% azure/latest-version %}}">Release Notes</a>]</sup>
+        </td>
+        <td>
+          Microsoft Azure
         </td>
         <td>
-          1.7.0 <sup>[<a href="https://www.deploykf.org/releases/version-matrix/#kubeflow-tools">Version Matrix</a>]</sup>
+          <a href="https://azure.github.io/kubeflow-aks/main">Website</a>
         </td>
       </tr>
       <tr>
-        <td>Kubeflow on Oracle Container Engine for Kubernetes</td>
-        <td>Oracle</td>
         <td>
-          Oracle Container Engine for Kubernetes (OKE)
+          VMware vSphere
         </td>
         <td>
-          <a href="https://github.com/oracle-devrel/kubeflow-oke">Website</a>
+          {{% vmware/latest-version %}}
         </td>
         <td>
-          1.6.0
+          VMware
         </td>
-      </tr>
-      <tr>
-        <td>Kubeflow on vSphere</td>
-        <td>VMware</td>
-        <td>VMware vSphere</td>
         <td>
           <a href="https://vmware.github.io/vSphere-machine-learning-extension/">Website</a>
         </td>
-        <td>
-          1.6.1
-        </td>
-      </tr>
-    </tbody>
-  </table>
-</div>
-
-### Legacy Distributions
```

Comment:
You have accidentally left all the legacy distributions (and placed them under the maintained table), if they aren't in a separate table, let's remove them.

We need to remove:

- Argoflow
- Kubeflow on OpenShift
- Arrikto Enterprise Kubeflow

Reply:
I will remove Argoflow + Arrikto, but for OpenShift can we can confirmation from the RedHat team @rimolive @terrytangyuan @rareddy @kubeflow/red-hat ?

Reply:
Red Hat is preparing to help with kubeflow 1.9 distribution testing. Let's keep it